asv_bench/benchmarks/categoricals.py

```diff
@@ -67,6 +67,24 @@ def time_existing_series(self):
         pd.Categorical(self.series)
 
 
+class AsType:
+    def setup(self):
+        N = 10 ** 6
+
+        self.df = pd.DataFrame(
+            np.random.default_rng()
+            .choice(np.array(list("abcde")), 4 * N)
+            .reshape(N, 4),
+            columns=list("ABCD"),
+        )
+
+        for col in self.df.columns:
+            self.df[col] = self.df[col].astype("category")
+
+    def astype_unicode(self):
```

Comment:
can you add benchmarks for other types of categories (int, dti) for example. show the results of the benchmarks.

Reply:
Ok!

Posted int benchmark in main thread + will add/post more

Reply:
right pls update these for int,float,string,datetime